modules/aws-waf/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "name" {
+  type    = string
+  default = "WAF"
```

Comment:
I believe it should not have default value to make sure that module users name resources properly

Reply:
Same thing as above, making this value null/empty makes it required but when preparing a template for prod/staging only staging will use this module, how can we solve this?

Reply:
I'd still keep it as required. In some projects we have only prod & non-prod AWS account separation and I can see a scenario in which we have such environments based on a single component:
* dev, where WAF is limited to devteam's ip(s)
* staging, where WAF is limited to devteam's and some additional client ip(s)
* specific-demo, where WAF is limited to devteam's and different additional ip(s)

Usually we also keep something like:
```tf
locals {
  name = "my-app-${var.environment}"
}
```

in infrastructure and we can simply pass it to aws-waf module as name. Then we always know that this specific WAF belongs to `my-app-dev` or `my-app-staging`, which makes it more clear.

Also, discussing this topic more deeply reminded me of really important, yet missing parameter - please add `tags` wherever possible 😉